benchmark/compare.js

```diff
@@ -69,10 +75,24 @@ if (showProgress) {
 
 (function recursive(i) {
   const job = queue[i];
-
-  const child = fork(path.resolve(__dirname, job.filename), cli.optional.set, {
-    execPath: cli.optional[job.binary],
-  });
+  const resolvedPath = path.resolve(__dirname, job.filename);
+
+  const cpuCore = cli.getCpuCoreSetting();
+  let child;
+  if (cpuCore !== null) {
+    const spawnArgs = ['-c', cpuCore, cli.optional[job.binary], resolvedPath, ...cli.optional.set];
+    child = spawn('taskset', spawnArgs, {
+      env: process.env,
+      stdio: ['inherit', 'pipe', 'pipe'],
+    });
+
+    child.stdout.pipe(process.stdout);
+    child.stderr.pipe(process.stderr);
+  } else {
+    child = fork(resolvedPath, cli.optional.set, {
+      execPath: cli.optional[job.binary],
+    });
+  }
 
   child.on('message', (data) => {
```

Comment:
seem like that this is not emitted when running with taskset

Reply:
https://nodejs.org/api/child_process.html#child_process_child_process_fork_modulepath_args_options:~:text=The%20child_process.fork,send()%20for%20details.

Reply:
you are right, I can add a fix for this in a seperate PR

Reply:
should we just run the compare script with taskset?

Reply:
I created this, let's see what @mcollina and @anonrig says
https://github.com/nodejs/performance/pull/162

Reply:
I don't understand what's the problem, please open an issue describing the full problem, reviewing merged prs is... not actionable.